src/platforms/javascript/guides/electron/index.mdx

```diff
@@ -88,24 +93,24 @@ The default transport automatically handles offline events caching. Following, a
 number of options that allow you to customize queueing behavior:
 
 ```javascript
-import * as Sentry from "@sentry/electron/main";
+import * as Sentry from '@sentry/electron/main';
 
 Sentry.init({
-  dsn: "___PUBLIC_DSN___",
+  dsn: '___PUBLIC_DSN___',
   transportOptions: {
     // The maximum number of days to keep an event in the queue.
     maxQueueAgeDays: 30,
     // The maximum number of events to keep in the queue.
     maxQueueCount: 30,
     // Called every time the number of requests in the queue changes.
-    queuedLengthChanged: (length) => { },
+    queuedLengthChanged: length => {},
     // Called before attempting to send an event to Sentry. Used to override queuing behavior.
     //
     // Return 'send' to attempt to send the event.
     // Return 'queue' to queue and persist the event for sending later.
     // Return 'drop' to drop the event.
-    beforeSend: (request) => isOnline() ? 'send' : 'queue'
-  }
+    beforeSend: request => (isOnline() ? 'send' : 'queue'),
```

Comment:
I don't think this is wrong, but this formatting doesn't feel conventional imo

Reply:
Yeah it's definitely not wrong, it's prettier conventional. It's wrapped like that for clarity, since otherwise it can be slightly unclear that it doesn't mean

```tsx
(request => isOnline()) ? 'send' : 'queue'`
```

Reply:
I agree though, unfortunately we probably don't want to turn off prettier just for stuff like this 😢